### ISSUE TITLE
Sticky hover style on touchscreens

### ISSUE THREAD

**Issue by sheenobu** [Alexays/Waybar#418]
This is related to #60 but it's actually sort of a touchscreen bug and persistent (until you manually hover/unhover via a mouse)

https://blog.usejournal.com/finally-a-css-only-solution-to-hover-on-touchscreens-c498af39c31c

> The problem arises when the user interacts with this element on a touchscreen: after the tap has been done, the hover effect is stuck on the element. This also occurs when the element is not even activated by the tapping, for instance if it was touched during scrolling.

I've tried disabling the hover via the guidance in #60 but it doesn't get any better. screenshot attached is using the standard style.css. Additionally, that pointer: fine suggestion is not supported on GTK CSS.

![2384c82c-b036-11e9-97d6-2816a867928b](https://user-images.githubusercontent.com/1443459/61990769-a7e58600-affb-11e9-9663-f5cdbaf9fa83.png)

**Comment by Alexays** [comment]
Yeah your css will fix the issue, but with that the hover effect is gone even with a mouse

**Comment by anonymous-user** [comment]
I am having the same issue on my XPS 13 also with a touchscreen. However I tried your code and it did not work 100%. I have set a custom background color for the current workspace and the hover effect remains there. Furthermore I noticed that hovering over non-active workspaces still shows a border effect on the button.
